components/create_post/create_post.jsx

```diff
@@ -434,26 +434,25 @@ export default class CreatePost extends React.Component {
             command === 'dnd' || command === 'offline';
     };
 
-    handleSubmit = (e) => {
+    handleSubmit = async (e) => {
         const {
             currentChannel: updateChannel,
             userIsOutOfOffice,
         } = this.props;
 
         if (this.props.isTimezoneEnabled) {
-            this.props.actions.getChannelTimezones(this.props.currentChannel.id).then(
-                (data) => {
-                    if (data.data) {
-                        this.setState({channelMembersCount: data.data.length});
-                    } else {
-                        this.setState({channelMembersCount: 0});
-                    }
-                }
-            );
+            const {data} = await this.props.actions.getChannelTimezones(this.props.currentChannel.id);
+            if (data) {
+                this.setState({channelTimezoneCount: data.length});
```

Comment:
Do we use this state property anywhere outside of this call chain? If we're going to fetch it each time we need it, it seems like we could avoid putting on the state at all and just pass it to `showNotifyAllModal` as a parameter.

Reply:
we use to render the message